src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Attr.java

```diff
@@ -5220,9 +5220,14 @@ public void visitAnnotatedType(JCAnnotatedType tree) {
     public void visitErroneous(JCErroneous tree) {
         if (tree.errs != null) {
             Env<AttrContext> errEnv = env.dup(env.tree);
```

Comment:
question: shouldn't we do: `env.dup(node, env.info.dup());` instead, that way both env's won't share the info?

Reply:
I was a little bit on the fence on this, as some code, like `Attr.attribClass` stores and re-sets the result, while other dupes the info. Let's dupe the info then. Thanks!

Reply:
I see, I didn't realize of the other pattern, not duping the info, we should probably standardize what we do at some point